.editorconfig

```diff
@@ -0,0 +1,6 @@
+[*.R]
```

Comment:
Intentional? (Wasn't in the PR description so I figured I'd ask)

Reply:
Yes, this just makes `.editorconfig` compatible editors use the right indent settings. I happened to be editing one of the .R files with one of those. :-)

https://editorconfig.org/